src/format.rs

```diff
@@ -281,6 +281,11 @@ impl FormatBuilder {
         FormatBuilder { format: Box::new(format) }
     }
 
+    /// Consumes the builder and returns the generated `TableFormat`
+    pub fn into_table_format(self) -> TableFormat {
+        *self.format
+    }
```

Comment:
You're probably right. I'll remove them. It's probably better (and easier) to add them later if required than remove them and break the API